### PR TITLE
assisted events scrape: add integration test

### PR DESCRIPTION
**Pull request by rccrdpccl**
Adds CI integration test for assisted events scrape

### CODE REVIEW

**Comment by osherdp** [comment]
/hold
(just to make sure it doesn't get in accidentally. rehearse jobs are optional)

**Comment by rccrdpccl** [comment]
Thanks, I need to remember this next time...

File: ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml
Comment (adriengentil) [inline ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml:50]: is the `sleep` required or is it just to debug?

**Review by adriengentil — COMMENTED** [review]
Nice :+1:

File: ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml
Comment (rccrdpccl) [reply ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml:50]: Just debug, I will need more changes but I want to close https://github.com/openshift-assisted/assisted-events-scrape/pull/74 first, so I can better test it here

**Comment by rccrdpccl** [comment]
Once this is merged rehearsal should work: https://github.com/openshift-assisted/assisted-events-scrape/pull/75

**Review by osherdp — APPROVED** [review]
Yep, looking good!

**Pull request merged**
